api/proto/teleport/legacy/types/types.proto

```diff
@@ -6159,6 +6161,57 @@ message PluginSCIMSettings {
   string default_role = 2;
 }
 
+// PluginAWSICSettings holds the settings for an AWS Identity Center integration.
+message PluginAWSICSettings {
+  option (gogoproto.equal) = true;
+
+  // IntegrationId is the Teleport OIDC integration used to gain access to the
+  // AWS account
+  string integration_id = 1;
```

Comment:
what's integration_id? Is it the name of the integration we create for AWS?

Reply:
Yes. The IDC plugin will use this integration to manipulate Identity Center, assuming whatever role the integration is configured with.